From 558bef1aa4bd75540202d4a0b2040ef9a4fb1fab Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Mon, 06 Jan 2025 10:02:34 +0200
Subject: [PATCH] [SW-208926] gaudi3: add bit for d2d fatal error

This patch adds d2d_fatal_err error bit in cold
reset flop, this will be used for handling d2d
fatal errors in d2d drtiver.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: I04cf0b336925ca909f57ec699f85774c970a4fe6
---

diff --git a/gaudi3/gaudi3_fw_if.h b/gaudi3/gaudi3_fw_if.h
index db21f3d..265212c 100644
--- a/gaudi3/gaudi3_fw_if.h
+++ b/gaudi3/gaudi3_fw_if.h
@@ -43,7 +43,8 @@
 			uint32_t pcie_bringup_by_erom : 1;
 			uint32_t wd_rst_cause_arcevent: 1;
 			uint32_t wd_rst_cause_arcpid: 1;
-			uint32_t reserved : 19;
+			uint32_t d2d_fatal_err: 1;
+			uint32_t reserved : 18;
 		};
 		uint32_t data;
 	};
